artifact/remote/git.go

```diff
@@ -38,7 +40,10 @@ func NewArtifact(rawurl string, c cache.ArtifactCache) (artifact.Artifact, func(
 		_ = os.RemoveAll(tmpDir)
 	}
 
-	return local.NewArtifact(tmpDir, c), cleanup, nil
+	// JAR/WAR/EAR doesn't need to be analyzed in git repositories.
+	disabled = append(disabled, analyzer.TypeJar)
```

Comment:
Just my preference:
I think it is better to implement this process in Trivy.

Reply:
Some analyzers should not be enabled regardless of callers. I think it is better to hard-code and disable them here.

Reply:
I got it.